vcpkg/produce/test-registry-ports-gha.md

```diff
@@ -0,0 +1,297 @@
+---
+title: Test your custom registry ports using vcpkg with GitHub Actions
+description: This tutorial shows users how to set up Continuous Integration environment for their registry's vcpkg ports using GitHub Actions.
```

Comment:
I don't think "Continuous Integration" needs to be capitalized here or elsewhere in the article.